rollup.config.js

```diff
@@ -8,7 +8,7 @@ const wrapeAndExportPlugin = require('./src/wrapeAndExportPlugin');
 const prepareForClientFunctionPlugin = require('./src/prepareForClientFunctionPlugin');
 
 module.exports = {
-  input: 'node_modules/@applitools/dom-snapshot/src/browser/processPageAndSerialize.js',
+  input: 'node_modules/@applitools/dom-snapshot/dist/processPageAndSerializeCjs.js',
```

Comment:
`/dist/processPageCjs` along with `serializeResources`

Reply:
FYI - the config as-is in the PR works.